pkg/chartutil/requirements.go

```diff
@@ -106,3 +116,138 @@ func LoadRequirementsLock(c *chart.Chart) (*RequirementsLock, error) {
 	r := &RequirementsLock{}
 	return r, yaml.Unmarshal(data, r)
 }
+
+// ProcessRequirementsConditions disables charts based on condition path value in values
+func ProcessRequirementsConditions(reqs *Requirements, cvals Values) {
+	var cond string
+	var conds []string
+
+	if reqs != nil && len(reqs.Dependencies) > 0 {
+		for _, r := range reqs.Dependencies {
+			var hasTrue, hasFalse bool
+			cond = string(r.Condition)
+			// check for list
+			if len(cond) > 0 {
+				if strings.Contains(cond, ",") {
+					conds = strings.Split(strings.TrimSpace(cond), ",")
+				} else {
+					conds = []string{strings.TrimSpace(cond)}
+				}
+				for _, c := range conds {
+					if len(c) > 0 {
+						// retrieve value
+						vv, err := cvals.PathValue(c)
+						if err == nil {
+							if vv.(bool) {
+								hasTrue = true
+							}
```

Comment:
Im glad you and inc0 reminded me about this part of code but for a different reason. I think it needs to look more like this or will fail if pathValue returns non bool (as far as I can tell)

````
if bv, ok := vv.(bool); ok{
        if bv == true {
			hasTrue = true
	}else{
			hasFalse = true
	}
}
````

If user sets a condition to a path that exists but resolves to a non-bool value, I suggest we either warn or ignore completely.  What do you think?